LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2020
+COPYRIGHT HOLDER: Division of Digital Psychiatry at Beth Israel Deaconess Medical Center
```

Comment:
We already have a LICENSE.md file? Is that eligible or is it a completely different format required here...?

Reply:
We need both. The .md is auto-generated I believe... Either way I used this as my example: https://github.com/tidyverse/dplyr.